src/main/java/com/example/pizzapp/mappers/CategoryMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.example.pizzapp.mappers;
+
+import com.example.pizzapp.dto.CategoryDto;
+import com.example.pizzapp.models.Category;
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+
+@Mapper(componentModel = "spring")
```

Comment:
можно сделать даже еще лучше вот так
```suggestion
@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
```